tests/test_admin_sync.py

```diff
@@ -406,6 +479,7 @@ def test_is_visible_method(client: TestClient) -> None:
     assert response.status_code == 200
     assert response.text.count('<span class="nav-link-title">Users</span>') == 1
     assert response.text.count('<span class="nav-link-title">Addresses</span>') == 1
+    assert response.text.count('<span class="nav-link-title">Profiles</span>') == 1
```

Comment:
Again I think we can avoid the tests in `test_admin_sync` and `test_admin_async`.

Reply:
:ok_hand: